src/Application/Actions/EmailVerificationToken/GetEmailVerificationTokenWithPersonId.php

```diff
@@ -10,11 +10,60 @@
 use Doctrine\ORM\EntityManagerInterface;
 use Doctrine\ORM\EntityRepository;
 use Laminas\Diactoros\Response\JsonResponse;
+use OpenApi\Annotations as OA;
 use Psr\Http\Message\ResponseInterface as Response;
 use Psr\Http\Message\ServerRequestInterface as Request;
 use Psr\Log\LoggerInterface;
 use Slim\Exception\HttpNotFoundException;
 
+/**
+ * // phpcs:disable -- OpenAPI needs long lines for good descriptions .
+ *  @OA\Get(
+ *     path="/v1/emailVerificationToken/{secret}/{personId}",
+ *     summary="Get an email verification token to check if its valid for an existng user account that does not yet have a password",
+ *     @OA\PathParameter(
+ *        name="personId",
+ *        description="UUID of the person",
+ *        @OA\Schema(
+ *        type="string",
+ *        format="uuid",
+ *        example="f7095caf-7180-4ddf-a212-44bacde69066",
+ *        pattern="[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}",
+ *        ),
+ *     ),
+ *      @OA\PathParameter(
+ *      name="secret",
+ *      description="secret code sent by email",
+ *      @OA\Schema(
+ *      type="string",
+ *      example="123456",
+ *      ),
+*      ),
+ *     @OA\Response(
+ *         response=200,
+ *         description="Token created and emailed to user, if they are NOT already registered exist, which the server
+ *          does not confirm or deny via http. If they are already registered then they are sent a message to remind
+ *          them of that.",
+ *         @OA\JsonContent(
+ *          @OA\Property(property="token", type="object", example={
+ *              "valid": true,
+ *              "email_address": "email@example.com",
+ *              "first_name": "Joe",
+ *              "last_name": "Bloggs",
+ *              }
```

Comment:
typing this only as "object" feels rather inadequate, but I couldn't quickly find how to declare the property types of an object inside a JsonContent with this annotation scheme. I followed examples like https://github.com/thebiggive/identity/blob/214450ac47e95bd06e05c8c8e237914e94dab1c6/src/Domain/Person.php#L98 which also just type as object with an example.

I've started avoiding returning meaningful objects directly as the full JSON doc in a response and wrapping them inside a JSON object like here to make space to add metadata or additional objects to the response in future without adding to the properties of the main thing i.e. the token here.